### ISSUE TITLE
"Table 'frag.viewer_project' doesn't exist error" on new stack instances

### ISSUE THREAD

**Issue by alanbchristie** [xchem/fragalysis-loader#19]
If I deploy the **Fragalysis Stack** (here I'm deploying the `latest` available today from Docker Hub) and then navigate to the stack's landing page *prior* to running a data loader container any subsequent execution of the **data loader** exhibits the following error: -

```
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/shell.py", line 101, in handle
    exec(sys.stdin.read())
  File "<string>", line 4, in <module>
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 464, in get_or_create
    return self.get(**lookup), False
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 374, in get
    num = len(clone)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 232, in __len__
    self._fetch_all()
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 1121, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
django.db.utils.ProgrammingError: (1146, "Table 'frag.viewer_project' doesn't exist")
```

I can rectify the situation and successfully run a loader by *bouncing* the Fragalysis Stack container with something like...

```
kubectl scale statefulset.apps/stack --replicas=0 
kubectl scale statefulset.apps/stack --replicas=1
```

With the stack *bounced* subsequent executions of the loader are successful.

The loader process is successful (in fresh/cold deployments) if I avoid navigating to the stack's landing page before running the first loader.

**Comment by reskyner** [comment]
I think this should be solved now - we're including migrations for the database at last. 

We can test this later when we build a dev environment for @Waztom

**Comment by alanbchristie** [comment]
Certainly when my scenario is re-run with stack/loader 2.0.3 this error isn't seen so I'll close this as '**fixed in 2.0.3**'.
